test/unit/test-performance.js

```diff
@@ -985,6 +988,41 @@ describes.realWin('PeformanceObserver metrics', {amp: true}, env => {
       });
       delete env.win.PerformanceEventTiming;
     });
+
+    it('created before performance service registered for Chrome 77', () => {
+      // Pretend that the EventTiming API exists.
+      PerformanceObserverConstructorStub.supportedEntryTypes = ['first-input'];
+
+      const entries = [
+        {
+          cancelable: true,
+          duration: 8,
+          entryType: 'first-input',
+          name: 'mousedown',
+          processingEnd: 105,
+          processingStart: 103,
+          startTime: 100,
+        },
+      ];
+      const getEntriesByType = env.sandbox.stub();
+      getEntriesByType.withArgs('first-input').returns(entries);
+      getEntriesByType.returns([]);
+      env.sandbox
+        .stub(env.win.performance, 'getEntriesByType')
+        .callsFake(getEntriesByType);
+
+      installPerformanceService(env.win);
+
+      const perf = Services.performanceFor(env.win);
+
+      expect(perf.events_.length).to.equal(1);
+      expect(perf.events_[0]).to.be.jsonEqual({
+        label: 'fid',
+        delta: 3,
+      });
+
+      delete env.win.PerformanceEventTiming;
```

Comment:
Not needed?

Reply:
Done in a8b3b8e8b0fe40cd95dea1f441d499dca8b995c8

Reply:
Done in a8b3b8e8b0fe40cd95dea1f441d499dca8b995c8